biggraphite/test_utils.py

```diff
@@ -207,7 +207,7 @@ def make_metric(self, name, metadata=None, **kwargs):
 @unittest.skipUnless(
     HAS_CASSANDRA, "CASSANDRA_HOME must be set to a >=3.5 install",
 )
-class TestCaseWithAccessor(TestCaseWithTempDir):
```

Comment:
Thinking about the future Cassandra + Elasticsearch driver, shouldn't we keep TestCaseWithAccessor.

Maybe the ES and Cassandra classes should not override TestCase's methods and thay should be called explicitly instead

Reply:
Composition is usually more effective than inheritance, so I would agree with that.
I'll abandon this PR.